src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -156,17 +111,38 @@ const ColonyHome = ({ match, location }: Props) => {
         <div className={styles.mainContent}>
           <ColonyTotalFunds colony={colony} />
           <Switch>
-            <Route path={COLONY_EVENTS_ROUTE} component={() => <>Events</>} />
             <Route
+              exact
+              path={COLONY_HOME_ROUTE}
+              component={() => <>Actions</>}
+            />
+            <Route
+              exact
+              path={COLONY_EVENTS_ROUTE}
+              component={() => <>Events</>}
+            />
+            <Route
+              exact
               path={COLONY_EXTENSIONS_ROUTE}
-              component={() => <>Extensions</>}
+              render={(props) => (
+                <Extensions {...props} colonyAddress={data.colonyAddress} />
+              )}
+            />
+            <Route
+              exact
+              path={[
+                COLONY_EXTENSION_DETAILS_ROUTE,
+                COLONY_EXTENSION_SETUP_ROUTE,
+              ]}
+              render={(props) => (
+                <ExtensionDetails
+                  {...props}
+                  colonyAddress={data.colonyAddress}
+                />
+              )}
             />
-            <Route path={COLONY_HOME_ROUTE} component={() => <>Actions</>} />
           </Switch>
         </div>
-        <aside className={styles.rightAside}>
-          <ColonyFunding colony={colony} currentDomainId={filteredDomainId} />
```

Comment:
Was this removed in error? I mean, we still use it :)

Reply:
It seems it's in `ColonyHomeLayout` now.

Reply:
Don't worry about this, I'll deal with rebasing and fixing the merge conflicts